bundles/media/imagine-adapter.rst

```diff
@@ -10,6 +10,12 @@ Available adapters are:
 * `imagick_extension`_
 * `vips_extension`_
 
+.. note::
+
+    You may require to install on your operation system ``libjpeg-dev``, ``libpng-dev``, ``libwebp-dev``, ``libavif-dev``, ...
+    or other libraries to support the range of image mime types you want to support. Linux distributions like
+    Alpine may are not shipping them in the base image.
+
```

Comment:
@Prokyonn may you have better suggestions here to make this kind of things easer to understand.